.nostr.local/settings.json

```diff
@@ -1,19 +1,70 @@
 {
   "info": {
-    "relay_url": "wss://nostream.localtest.me",
-    "name": "nostream.localtest.me",
-    "description": "A nostr relay written in TypeScript.",
-    "pubkey": "replace-with-your-pubkey",
+    "relay_url": "wss://nostr-relay-dev.wlvs.space",
```

Comment:
will revert once I'm done testing
this file should probably not be committed anyway?